packages/config/config/devices/templates/master_template.json

```
@@ -403,6 +426,33 @@
 			}
 		]
 	},
+	"smart_switch_mode_0-3": {
+		"label": "Smart Switch Mode",
+		"valueSize": 1,
+		"minValue": 0,
+		"maxValue": 3,
+		"defaultValue": 3,
+		"unsigned": true,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "Disable",
+				"value": 0
+			},
+			{
+				"label": "Local control only",
+				"value": 1
+			},
+			{
+				"label": "Central scene only",
```

```suggestion
				"label": "Central Scene only",
```
This is a CC name, so we should capitalize it.